contracts/governance/gov_constants.cairo

```diff
@@ -0,0 +1,10 @@
+const PROPOSAL_VOTING_TIME_BLOCKS = 50; // TODO what is the blocktime on mainnet precisely? 30 mins? any commitment from Starkware? // TODO make higher after testing
+const NEW_PROPOSAL_QUORUM = 200; // 1/200 of totalSupply required to propose an upgrade. Quorums don't take into account investors. at all, they don't count into total eligible voters, but do vote.
+const QUORUM = 10; // 1/20 of totalSupply required to participate or pass
+const TEAM_TOKEN_BALANCE = 5000000000000000000; // 5 * 10^18. Used only in calculation of investor voting power.
```

Comment:
what does this number represent?

Reply:
Investor voting power is totalSupply - TEAM_TOKEN_BALANCE, will be roughly equal to a quarter of voting power, but never will be along with team more than 1/2